pyqtgraph/graphicsItems/ViewBox/ViewBox.py

```diff
@@ -1293,8 +1341,15 @@ def wheelEvent(self, ev, axis=None):
             return
 
         s = 1.02 ** (ev.delta() * self.state['wheelScaleFactor']) # actual scaling factor
+
+        if s < 1.02 and any(self._reachedMaxZoom):
+            print("Can't zoom further")
+            return
+
         s = [(None if m is False else s) for m in mask]
-        center = Point(fn.invertQTransform(self.childGroup.transform()).map(ev.pos()))
+        inv = fn.np_invert_qtransform(self.childGroup.transform())
```

Comment:
not using `invertQTransform` here is going to have a _significant_ performance penalty, why is the numpy version needed here?

Reply:
Well, if I have tested it correctly, it breaks, if you don't use it there.  Every mapping with `invertQTransform` will break with Infinities and NaNs popping up after a certain threshhold.  Also the mapping with Qt using `tr.map()` will break if the values are near sys.float_max, so I have to both use np_invert and np_map, so it doesn't break. 

I didn't notice any differences in speed, but I also didn't check for them. It would makea lot of sense for the Qt versions to be faster, as they use the C++ implementation, but I would also argue, that the numpy versions are not that far off, as e.g. the inversion algorithm is heavily optimized as well.  The mapping function np_map on the other hand should be (much) slower than Qt's map function, but that would also need to be tested. One way to fix possible performance issue is to use the numpy variant of inverting and mapping only after the zoom has reached certain threshold (e.g. over 1e304).